examples/stats/plot_regress_continuous.py

```diff
@@ -0,0 +1,59 @@
+"""
+========================================
+Regression on continuous data (rER[P/F])
+========================================
+
+This demonstrates how rERPs/regressing the continuous data is a
+generalisation of traditional averaging. If all preprocessing steps
+are the same and if no overlap between epochs exists and if all
+predictors are binary, regression is virtually identical to traditional
+averaging.
+If overlap exists and/or predictors are continuous, traditional averaging
+is inapplicable, but regression can estimate, including those of
+continuous predictors.
+
+rERPs are described in:
```

Comment:
does it render like this? http://martinos.org/mne/dev/auto_examples/preprocessing/plot_interpolate_bad_channels.html#references